contents/docs/session-replay/network-recording.mdx

```diff
@@ -6,119 +6,33 @@ availability:
-            request.name = request.name.split('?')[0]
-
-            // ...redact the request or response body **however makes sense for your payloads**
-            request.requestBody = request.requestBody?.replace(/"password":\s*"[^"]*"/g, '"password": "redacted-password"')
-            request.responseBody = request.responseBody?.replace(/"password":\s*"[^"]*"/g, '"password": "redacted-password"')
-
-            // ... or remove the request body
-            request.requestBody = undefined
-
-            // ... and more
-
-            // finally return the request
-            return request
-        }
-    }
-})
-```
-
-## Troubleshooting
-
-### Recording from localhost
-
-Due the very high volume of network requests that some tools can make (for example when running hot-reload during development) PostHog does not capture network requests when running on localhost
-
-### Requests early in the page lifecycle don't capture all information
-
-PostHog has to wrap `fetch` and `xhr` in order to capture network requests. If your application makes network requests before PostHog has had a chance to wrap them, then PostHog will not capture all information about the request.
-
-### PostHog truncated the request or response body
-
-In order to maintain service levels we truncate all request and response bodies at 1MB
-
-### A network request says it was redacted
-
-We are cautious in what we capture. If you think we're being too cautious you can override the masking function.
-
-### I want to query the network performance data I capture
-
-We'd love that too! It's not possible right now but watch this space. You can [subscribe for updates in GitHub](https://github.com/PostHog/posthog/issues/19686)
+import Tab from "components/Tab"
+import WebNetwork from "./_snippets/web-network.mdx"
+import AndroidNetwork from "./_snippets/android-network.mdx"
+import IOSNetwork from "./_snippets/ios-network.mdx"
+import ReactNativeNetwork from "./_snippets/react-native-network.mdx"
+
+Session replay allows you to capture network requests and responses, providing insights into network performance and potential issues. This feature can be particularly useful for debugging and optimizing your application's network interactions.
+
+<Tab.Group tabs={['Web', 'Android', 'iOS']}>
+    <Tab.List>
+        <Tab>Web</Tab>
+        <Tab>Android</Tab>
+        <Tab>iOS</Tab>
+    </Tab.List>
+    <Tab.Panels>
+        <Tab.Panel>
+            <WebNetwork />
+        </Tab.Panel>
+        <Tab.Panel>
+            <AndroidNetwork />
+        </Tab.Panel>
+        <Tab.Panel>
+            <IOSNetwork />
+        </Tab.Panel>
+    </Tab.Panels>
+</Tab.Group>
+
+### Performance considerations
+
+Capturing network performance data can have an impact on your application's performance. Be mindful of the amount of data you're capturing and consider implementing sampling or other optimization techniques if you're dealing with high-volume network traffic.
```

Comment:
@daniloc a customer is asking about this advice... do you know the context for introducing it - it seems new to this PR